packages/web3torrent/src/library/web3torrent-lib.ts

```diff
@@ -346,19 +360,37 @@ export default class WebTorrentPaidStreamingClient extends WebTorrent {
       await this.closeDownloadingChannels(torrent);
     });
 
-    torrent.on(TorrentEvents.ERROR, err => {
-      log.error({err, torrent}, 'Torrent ERROR: ');
-      this.emit(ClientEvents.TORRENT_ERROR, {torrent, err});
+    torrent.on(TorrentEvents.ERROR, error => {
+      log.error('Torrent ERROR: ', torrent, error);
+      if (error && (error as Error).message.includes('Cannot add duplicate torrent')) {
+        this.resumeTorrent(torrent.infoHash); // horrible way of doing things, will do until we make pause/resume
```

Comment:
this is really, really messy.
I promise that I will fix this one in another PR. The idea with this PR was to make an initial approach to pausing too. To see what could be happening, and make it work semi-okay